webapp/pages/index.vue

```diff
@@ -10,7 +10,7 @@
         />
       </ds-flex-item>
       <hc-post-card
-        v-for="(post, index) in uniq(Post)"
+        v-for="(post, index) in posts"
```

Comment:
maybe I need to add this `uniq` back??

Reply:
I dunno, what could add a duplicate post here?